### YamlMime:ManagedReference
items:
- uid: v3.2.0.3:Xunit.Sdk.IAssertionException
  commentId: T:Xunit.Sdk.IAssertionException
  id: IAssertionException
  parent: v3.2.0.3:Xunit.Sdk
  children: []
  langs:
  - csharp
  - vb
  name: IAssertionException
  nameWithType: IAssertionException
  fullName: Xunit.Sdk.IAssertionException
  type: Interface
  source:
    remote:
      path: Sdk/IAssertionException.cs
      branch: d61847f1ba2595c0cad96f57fc778a03ee292506
      repo: https://github.com/xunit/assert.xunit
    id: IAssertionException
    path: ../src/xunit.v3.assert/Asserts/Sdk/IAssertionException.cs
    startLine: 17
  assemblies:
  - xunit.v3.assert
  namespace: Xunit.Sdk
  summary: >-
    This is a marker interface implemented by all built-in assertion exceptions so that

    test failures can be marked with <xref href="v3.2.0.3:Xunit.Sdk.FailureCause.Assertion" data-throw-if-not-resolved="false"></xref>.
  example: []
  syntax:
    content: public interface IAssertionException
    content.vb: Public Interface IAssertionException
references:
- uid: v3.2.0.3:Xunit.Sdk.FailureCause.Assertion
  commentId: F:Xunit.Sdk.FailureCause.Assertion
  isExternal: true
- uid: v3.2.0.3:Xunit.Sdk
  commentId: N:Xunit.Sdk
  href: Xunit.html
  name: Xunit.Sdk
  nameWithType: Xunit.Sdk
  fullName: Xunit.Sdk
  spec.csharp:
  - uid: v3.2.0.3:Xunit
    name: Xunit
    href: Xunit.html
  - name: .
  - uid: v3.2.0.3:Xunit.Sdk
    name: Sdk
    href: Xunit.Sdk.html
  spec.vb:
  - uid: v3.2.0.3:Xunit
    name: Xunit
    href: Xunit.html
  - name: .
  - uid: v3.2.0.3:Xunit.Sdk
    name: Sdk
    href: Xunit.Sdk.html
memberLayout: SeparatePages
